FlowCrypt/ObjC/ObjException.m

```diff
@@ -0,0 +1,26 @@
+//
+//  ObjException.m
+//  FlowCrypt
+//
+//  Created by  Ivan Ushakov on 25.01.2022
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+#import "ObjException.h"
+
+@implementation ObjException
```

Comment:
It'll be better to call it `ObjcException`, as `ObjException` doesn't make it clear that we're catching exceptions from Objective C.